lib/flutter_config.dart

```diff
@@ -5,7 +5,7 @@ import 'package:flutter/services.dart';
 /// and as a `NSDictionary` for iOS
 class FlutterConfig {
   /// An instance of all environment variables
-  Map<String, dynamic> _variables;
```

Comment:
Hi, I think it might be better to annotate this as `late` instead of assigning it right away. Since `loadEnvVariables()` needs to be called in order to load the variable from the native platform, it's makes since to assign it there

Reply:
Thanks! PR Updated.